pkg/metallb/frr_test.go

```diff
@@ -0,0 +1,400 @@
+package metallb
+
+import (
+	"testing"
+	"time"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/schemes/metallb/frrtypes"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	frrConfigurationGVK = schema.GroupVersionKind{
```

Comment:
add a scheme attachers list then you can use `clients.TestClientParams{SchemeAttachers: frrTestSchemes}`
```go
frrTestSchemes := []clients.SchemeAttacher{
    frrTypes.AddToScheme,
}
```

Reply:
added